adapters/discord/src/types/application-role-connection.ts

```diff
@@ -0,0 +1,62 @@
+import { Internal, Locale } from "."
+
+export namespace ApplicationRoleConnection {
+  /** https://discord.com/developers/docs/resources/application-role-connection-metadata#application-role-connection-metadata-object-application-role-connection-metadata-structure */
+  export interface Metadata {
+    /** type of metadata value */
+    type: MetadataType
+    /** dictionary key for the metadata field (must be a-z, 0-9, or _ characters; 1-50 characters) */
+    key: string
+    /** name of the metadata field (1-100 characters) */
+    name: string
+    /** translations of the name */
+    name_localizations?: Record<Locale, 'string'>
```

Comment:
```suggestion
    name_localizations?: Record<Locale, string>
```

这里肯定不是 `'string'` 吧？